docs/helm-deployment-aws_cloud.md

```diff
@@ -402,6 +402,31 @@ helm install alfresco-incubator/alfresco-content-services \
 
 **Note:** When using an external database, make sure that the database is reachable by the K8S nodes and is up and ready.
 
+### Using the Alfresco S3 Connector
+
+If you have the S3 connector amp applied to your docker image then you could enable it in kubernetes by setting the following values:
+
+```
+helm install alfresco-incubator/alfresco-content-services \
+--set externalProtocol="https" \
+--set externalHost="$EXTERNALHOST" \
+--set externalPort="443" \
+--set repository.adminPassword="$ALF_ADMIN_PWD" \
+--set alfresco-infrastructure.persistence.efs.enabled=true \
```

Comment:
I suspect we are leaving EFS for DB and Solr, but those are not clustered. Increasing the number of desired pods will not make them clustered as well. Should EFS be removed in this example and default EBS will be used?

Reply:
S3 is not a default so using EFS still stands for clustering / DB and Solr.

Reply:
Yes, Alex i would still leave it like this, with EFS, since EBS has a lot of issues moving from one node to another in case one of the pods gets rescheduled. But you could without issues do 
 --set alfresco-infrastructure.enabled=false to go for the default gp2.